years.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if (grep -r "Copyright[[:space:]]\+(c)[[:space:]]\+2009-.*rultor.com" --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . | grep -v 2009-`date +%Y`) then
```

Comment:
@vkuchyn but he just wrote
> just be careful with white-space

I don't see anything about using  `[[:space:]]` instead of  `' '`